safe/gui/tools/wizard_metadata.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool by AusAid **GUI InaSAFE Wizard Metadata.**
+
+ ** NOTE This file meant for temporary storing metadata to be later moved
+         to proper places.
+
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+__author__ = 'qgis@borysjurgiel.pl'
+__revision__ = '$Format:%H$'
+__date__ = '12/04/2016'
+__copyright__ = ('Copyright 2012, Australia Indonesia Facility for '
+                 'Disaster Reduction')
+
+
+road_class_mapping = [{
```

Comment:
Yes, I agree with @Gustry 
We need two thing for each class, `key` and `name`. `key` is the machine-friendly, not translatable. Where `name` is the human friendly one, it should be translatable.

Please take a look at the example below from class mapping in definitions.py

```
    'classes': [
        {
            'key': 'wet',
            'name': tr('wet'),
            'description': tr('Water above ground height.'),
            'numeric_default_min': 1,
            'numeric_default_max': 9999999999,
            'optional': True
        },
        {
            'key': 'dry',
            'name': tr('dry'),
            'description': tr('No water above ground height.'),
            'numeric_default_min': 0,
            'numeric_default_max': (1 - small_number),
            'optional': True
        }
    ]
```
A description also should be added.

I am fine with `value_mapping`. What do you think @timlinux